limbus/core/param.py

```diff
@@ -513,7 +534,7 @@ async def send(self) -> None:
             dst_param: Param = ref.param
             assert isinstance(dst_param, InputParam)  # they must be of type InputParam
             assert dst_param._parent is not None
-            if not dst_param._parent._is_running:
+            if not _check_asyncio_task_exists(dst_param._parent):
                 # start the execution of the component if it is not started yet
                 asyncio.create_task(dst_param._parent.execute())
```

Comment:
```suggestion
    if self.async_task is not None:
                self.async_task = asyncio.create_task(dst_param._parent.execute())
```
where `self.async_task: Optional[asyncio.Task] = None`
i prefer to have control over the task

Reply:
taht approach is not gonna work, several params can create the same component. As far as I know, to know if the task exists the only way is to check the global list of tasks in the running loop.